### ISSUE TITLE
Provide alternatives to other services from SF

### ISSUE THREAD

**Issue by paulolieuthier** [helb/helb.github.io#6]
Sourceforge doesn't offer code and file hosting only. It also offers bug trackers and mailing lists. Unfortunately, AFAIK, there is not a single full replacement for it. Please, show alternatives for all services.

**Comment by peterrecore** [comment]
Binary distribution is another service that probably merits a column in the chart.

**Comment by ihamburglar** [comment]
Yeah when I think about binary distribution I don't think GitHub or Launchpad.  Downloading a zip of master, is not binary distribution.

**Comment by paulolieuthier** [comment]
I don't know about Launchpad, but Github allows downloading binaries for each release.

**Comment by baweaver** [comment]
Bitbucket and Github at least have issue tracking. Bitbucket can be tied into most everything Atlassian, so it has a lot more powerful tools on that end you can leverage such as Jira.

Binary distribution though, I'd be interested to hear of other sources. We could ping the Github team and ask them if they'd consider it as a feature.

**Comment by ihamburglar** [comment]
Github is built for developers.  Unless a project makes use of github pages or has an external site, the default interface for getting stuff is developer friendly, user unfriendly.

**Comment by jubalh** [comment]
like @paulolieuthier said, you can upload binaries for each release. it doesnt have to be autogenerated on a certain git hash. can be custom.

**Comment by comuttun** [comment]
Could you consider add [osdn.jp](http://osdn.jp/) ?

Previously, the site was sourceforge.jp.
On May 11, they had thrown away "Sourceforge" and "Slashdot" brand names.

see. [Press releases translated by Google Translate](https://translate.google.com/translate?hl=ja&sl=ja&tl=en&u=http%3A%2F%2Fosdn.co.jp%2Fpress%2F2015%2F04%2F%E3%82%B5%E3%82%A4%E3%83%88%E5%90%8D%E7%A7%B0%E5%A4%89%E6%9B%B4%E3%81%AE%E3%81%8A%E7%9F%A5%E3%82%89%E3%81%9B)

osdn.jp provides features same as sourceforge.net, and also provides chamber (not project but per user repository).
